### DISCUSSION TITLE
data exceeds 1273 characters : chillerlan\QRCode\Data\QRCodeDataException

### DISCUSSION THREAD

**Discussion by abhijeetavion** [chillerlan/php-qrcode#192] [General]
Hello Team,

Getting following error when generating QR code

'chillerlan\QRCode\Data\QRCodeDataException   
data exceeds 1273 characters'

i set the version option to  QRCode::VERSION_AUTO  but still same error 

Please Help

**Comment by codemasher** [comment]
That sounds like the string you're trying to encode exceeds the maximum length for a binary string at version 40 and ECC level H (using logo space i assume?). See http://www.qrcode.com/en/about/version.html

https://github.com/chillerlan/php-qrcode/blob/284c42db884b98b2453e39a7e10a7a04239d0019/tests/QRMaxLengthTrait.php#L69
